std_msgs/Header header

uint32 image_width
uint32 image_height

FaceDetection[] faces
